frontend/src/Frontend/ChainwebApi.hs

```diff
@@ -63,6 +71,10 @@ serviceBaseUrl :: ChainwebHost -> Text
 serviceBaseUrl (ChainwebHost nc cver) =
     hostToText (_netConfig_serviceHost nc) <> "/chainweb/0.0/" <> cver <> "/"
 
+dataBaseUrl :: ChainwebHost -> Text
+dataBaseUrl (ChainwebHost nc _) =
+    maybe "https://estats.chainweb.com" hostToText (_netConfig_dataHost nc)
+
```

Comment:
```suggestion
```
I think we should remove this definition, because we don't want to use a default CW-D URL when it's absent in the configuration, because; 1) A given CW-D instance serves a single Kadena network 2) In other pages, like event search, `block-explorer` treats the absence of a CW-D gracefully by displaying a message that informs the user about the lack of support for that page.